<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HashcionaryClass</class>
 <widget class="QMainWindow" name="HashcionaryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hashcionary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="linhaBusca">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Digite a palavra.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoBuscar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans Semibold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hashcion√°rio</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="browserSignificado">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Significado.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionAtiva">
   <property name="text">
    <string>ativa</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Hashcionary.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>botaoBuscar</sender>
   <signal>clicked()</signal>
   <receiver>linhaBusca</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
